tutorials/vr/vr_starter_tutorial/vr_starter_tutorial_part_one.rst

```diff
@@ -0,0 +1,1146 @@
+an ARVR viewport.
+
+If the VR interface initialized successfully, we then get the root :ref:`Viewport <class_Viewport>` and set the `arvr` property to ``true``. This will tell Godot to use the initialized
+ARVR interface to drive the :ref:`Viewport <class_Viewport>` display. After setting the ``arvr`` property to ``true``, we set the ``hdr`` property to ``false``. We do this because
+most of the VR headsets do not currently support HDR rendering.
+
+Finally, we disable VSync so the Frames Per Second (FPS) is not capped by the computer monitor. After this we tell Godot to render at ``90`` frames per second, which is the
+standard for most VR headsets. Without disabling VSync, the normal computer monitor may limit the frame rate of the VR headset to the frame rate of the computer monitor.
+
+.. note:: In the project settings, under the ``Physics->Common`` tab, the physics FPS has been set to ``90``. This makes the physics engine run at the same frame rate as
+          the VR display, which makes physics reactions look smoother when in VR.
+
+_________________
+
+That is all we need to do for Godot to launch OpenVR within the project! Go ahead and give it a try if you want. Assuming everything works, you will be able to look around
+the world. If you have a VR headset with room tracking, then you will be able to move around the scene within the limits of the room tracking.
+
+Creating the controllers
+------------------------
+
+.. image:: img/starter_vr_tutorial_hands.png
+
+Right now all that the VR user can do is stand around, which isn't really what we are going for unless we are working on a VR film. Lets write the code for the
+VR controllers. We are going to write all of the code for the VR controllers in one go, so the code is rather long. That said, once we are finished you will be
+able to teleport around the scene, artificially move using the touchpad/joystick on the VR controller, and be able to pick up, drop, and throw
+:ref:`RigidBody <class_RigidBody>`-based nodes.
+
+First we need to open the scene used for the VR controllers. ``Left_Controller.tscn`` or ``Right_Controller.tscn``. Let's briefly go over how the scene is setup.
+
+How the VR controller scene is setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In both scenes the root node is a ARVRController node. The only difference is that the ``Left_Controller`` scene has the ``Controller Id`` property set to ``1`` while
+the ``Right_Controller`` has the ``Controller Id`` property set to ``2``.
+
+Next is the ``Hand`` :ref:`MeshInstance <class_MeshInstance>` node. This node is used to display the hand mesh that will be used when the VR controller is not holding onto a
+:ref:`RigidBody <class_RigidBody>` node. The hand in the ``Left_Controller`` scene is a left hand, while the hand on the ``Right_Controller`` scene is a right hand.
+
```

Comment:
We should add a footnote here that the ARVR system attempts to use these two ID's. For Rift/Index/MR this generally works as the controllers identify themselves as left and right and don't support additional controllers. But Vive wands do not. We do ensure in the latest version that trackers won't take slots 1 and 2.